callstats-java-sdk/src/test/java/io/callstats/sdk/CallStatsTest.java

```diff
@@ -76,71 +75,72 @@ public void initializeTest() {
 		Mockito.verify(listener).onInitialized(msg);
 	}
 	
-	/**
-	 * Initialize with invalid app id test.
-	 */
-	@Test
-	public void initializeWithInvalidAppIdTest() {
-		CallStatsErrors error = CallStatsErrors.HTTP_ERROR;
-		String errMsg = "SDK Authentication Error";
-
-		callstatslib.initialize(appId+1, appSecret, "jit.si.345",serverInfo,listener);
-
-		try {
-			Thread.sleep(2000);
-		} catch (InterruptedException e) {
-			// TODO Auto-generated catch block
-			e.printStackTrace();
-		}
-		Mockito.verify(listener).onError(error, errMsg);
-	}
-	
-	/**
-	 * Initialize with null args test.
-	 */
-	@Test
-	public void initializeWithNullArgsTest() {
-		Throwable e = null;
-
-		try {
-
-		callstatslib.initialize(0, appSecret, "jit.si.345",serverInfo,listener);
-
-		} catch (Throwable e1) {
-			e = e1;
-		}
-		assertTrue(e instanceof IllegalArgumentException);
-		
-		try {
-
-			callstatslib.initialize(appId, appSecret, null,serverInfo,listener);
-
-		} catch (Throwable e1) {
-				e = e1;
-		}
-		assertTrue(e instanceof IllegalArgumentException);
-		
-		try {
-
-			callstatslib.initialize(appId, "", "jit.si.345",serverInfo,listener);
-
-		} catch (Throwable e1) {
-				e = e1;
-		}
-		assertTrue(e instanceof IllegalArgumentException);
-		
-		try {
-			callstatslib.initialize(appId, appSecret, "",serverInfo,listener);
-
-		} catch (Throwable e1) {
-				e = e1;
-		}
-		assertTrue(e instanceof IllegalArgumentException);
-	}
+//	/**
```

Comment:
Remove instead of commenting if this is not intended to be used. Or uncomment and use `@Ignore` instead of `@Test` if you want to simply ignore it for now.